### PR TITLE
add ml image to gcp

### LINKED ISSUE #700: Bring ML image to gcp cluster?
The AWS cluster has a "ML Notebook" image, which points to https://github.com/pangeo-data/pangeo-docker-images/tree/master/ml-notebook. The GCP cluster does not. But many GCP users (e.g. @anirban89) want to do ML from GCP.

Is it as simple as adding this image to the profile list?

### PR DESCRIPTION
**Pull request by rabernat**
closes #700 

cc @anirban89

### CODE REVIEW

File: deployments/gcp-uscentral1b/config/common.yaml [hunk 1]
```diff
@@ -75,6 +75,17 @@ daskhub:
                     'mem_guarantee': '12G',
                 }
             },
+            {
+                'display_name': 'ML Image - Large (12 GB - 16 GB)',
```
Comment (TomAugspurger) [inline deployments/gcp-uscentral1b/config/common.yaml:79]: I think this doesn't align with the memory limits below.

**Comment by TomAugspurger** [comment]
Thanks!

**Comment by rabernat** [comment]
Ok this works. But I realized that the ml image doesn't have Keras! 🤦

**Comment by TomAugspurger** [comment]
In the meantime, @anirban89, does using `tf.keras` work for you?

**Comment by rabernat** [comment]
Actually keras works. You just have to import it from tensorflow!

```python
from tensorflow import keras
```

**Pull request merged**